docs/UserGuide.md

```diff
@@ -16,29 +16,30 @@ title: User Guide
 
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your computer.
+1. Ensure you have Java `11` installed in your computer.
 
 1. Download the latest `productiv.jar` from [here](https://github.com/AY2021S1-CS2103T-F11-2/tp/releases).
 
 1. Copy the `.jar` file to your preferred folder.
 
 1. Double-click the file to start *Productiv*. 
-Your `dashboard` should appear in a few seconds. 
+Your dashboard should appear in a few seconds. 
 Note that the app contains some sample data.<br>
 
    ![Ui](images/Ui.png)
    <figcaption>Dashboard</figcaption><br>
    
 1. Type a command in the command box and press Enter to execute it. 
-Here are some example commands you can try:
+Here is a sequence of example commands you can try:<br>
 
-   * **`switch`**`dv` : Switches to deliverable mode.
+   1. **`switch`** `dv` : Switches to deliverable mode.
 
-   * **`add`**`r/stk n/Betty Clarke e/betty@email.com` : If in contact mode, adds a contact named `Betty Clarke`.
+   1. **`add`** `t/Find profile page template m/2.1.1 by/12-12-2020 12:00` : Adds a deliverable with the 
+   title `Find profile page template`, milestone `2.1.1` and deadline `12-12-2020 12:00`.
 
-   * **`delete`**`1` : If in meeting mode, deletes the 1st meeting shown.
+   1. **`delete`** `1` : Deletes the 1st deliverable shown.
```

Comment:
just in case, maybe better to leave the "if in .. mode"

Reply:
I feel like its alright, actually if i wanted to really include I should be including that condition for "add t/deliverable..." instead

Reply:
right, ok just saw the 'sequence'